python/test/test_action_yml.py

```diff
@@ -1,17 +1,20 @@
+import os
+import pathlib
 import unittest
 
 import yaml
-from yaml import Loader
+
+project_root = pathlib.Path(os.path.dirname(__file__)) / '../..'
```

Comment:
```suggestion
project_root = pathlib.Path(os.path.dirname(__file__)) / '..' / '..'
```

Reply:
I'm going for the suggestion with `.parent` for each 3 files